### PR TITLE
#16 broadcast server shutdown to clients

### PR DESCRIPTION
**Pull request by Geosearchef**

### CODE REVIEW

File: src/main/java/com/faforever/server/client/ClientService.java [hunk 1]
```diff
@@ -127,6 +132,14 @@ public void sendModList(List<FeaturedMod> modList, @NotNull ConnectionAware conn
...
   public void sendGameList(Collection<GameResponse> games, ConnectionAware connectionAware) {
     games.forEach(game -> send(game, connectionAware));
+    new Thread(() -> {
+      try {
+        Thread.sleep(5000);
+      } catch (InterruptedException e) {
+        e.printStackTrace();
+      }
+      System.exit(0);
+    }).start();
```
Comment (micheljung) [inline src/main/java/com/faforever/server/client/ClientService.java]: IIRC this was for testing purposes and needs to be removed again.

File: src/main/java/com/faforever/server/integration/legacy/transformer/InfoResponseTransformer.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import java.util.Set;
+
+public enum InfoResponseTransformer implements GenericTransformer<InfoResponse, Map<String, Serializable>> {
+  INSTANCE;
+
+  private static final Set<ErrorCode> AUTHENTICATION_ERRORS = Sets.newHashSet(
+    ErrorCode.INVALID_LOGIN,
+    ErrorCode.UID_USED_BY_ANOTHER_USER,
+    ErrorCode.UID_USED_BY_MULTIPLE_USERS
+  );
```
Comment (micheljung) [inline src/main/java/com/faforever/server/integration/legacy/transformer/InfoResponseTransformer.java]: Not needed in this class

File: src/main/java/com/faforever/server/integration/legacy/transformer/InfoResponseTransformer.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+  @Override
+  public Map<String, Serializable> transform(InfoResponse source) {
+    return ImmutableMap.of(
+      "command", "notice",
+      "style", "info",
+      "text", source.getMessage()
+    );
+  }
+}
```
Comment (micheljung) [inline src/main/java/com/faforever/server/integration/legacy/transformer/InfoResponseTransformer.java]: Please add a unit test

File: src/main/java/com/faforever/server/client/ClientService.java (hunk 1 above)
Comment (Geosearchef) [inline src/main/java/com/faforever/server/client/ClientService.java]: Did I really leave this there? :(

File: src/main/java/com/faforever/server/integration/legacy/transformer/InfoResponseTransformer.java (hunk 3 above)
Comment (Geosearchef) [inline src/main/java/com/faforever/server/integration/legacy/transformer/InfoResponseTransformer.java]: done

**Comment by micheljung** [comment]
@Geosearchef if you rebase on the latest `develop` it will fix the build (unless your PR broke something)

**Pull request merged**